pantos/common/configuration.py

```diff
@@ -19,13 +19,15 @@
 
 # Ordered by priority
 _CONFIGURATION_PATHS = [
-    pathlib.Path.cwd(),
     pathlib.Path.home(),
     pathlib.Path.home() / '.config',
     pathlib.Path('/etc/pantos'),
     pathlib.Path('/etc')
 ]
```

Comment:
Alternative
```
_CONFIGURATION_PATHS = [ pathlib.Path.cwd() if pathlib.Path.cwd() != pathlib.Path('/') else None ] + [
    pathlib.Path.home(),
    pathlib.Path.home() / '.config',
    pathlib.Path('/etc/pantos'),
    pathlib.Path('/etc')
]
```

Reply:
Hmm, any option is ugly IMO, I will just merge like this 👍